.taskcluster.yml

```diff
@@ -49,6 +49,23 @@ tasks:
                   else:
                       $if: 'tasks_for in ["cron", "action"]'
                       then: '${repository.project}'
+          base_branch:
```

Comment:
Shouldn't this also be called `base_ref` as it's not necessarily going to be a branch?

Reply:
Good call, fixed! I also renamed `head_branch` into `head_ref`.